Updated obsolete cloud-related terms (#4616)
SkylineCommunications/dataminer-docs: user-guide/Advanced_Functionality/DataMiner_Agents/Upgrading_a_DMA/Upgrading_a_DataMiner_Agent_in_the_Update_Center.md

   > - *Modules* > *System* *configuration* > *Agents* > *Upgrade / restore*
   > - *Modules* > *System* *configuration* > *Agents* > *Stop*

> 1. Enter your dataminer.services credentials when you are prompted to do so.

Reviewer: As this section is related to old DataMiner versions, the UI will still ask for "DCP" credentials. As such, to prevent confusion, I'd change this as follows:
```suggestion
1. When you are prompted to enter DCP credentials, enter your credentials for dataminer.services.
```
